civihr_default_theme/scss/base/_variables.scss

```diff
@@ -173,3 +173,16 @@ $crm-customselect-padding-right: $crm-customselect-caret-width + 12 !default;
 $input-border:              $crm-color-neutral !default;
 
 $crm-ui-select-top-padding: 7px;
+
+
+//== User Menu
+//
+//##
+$chr-user-menu-color: $light-pale;
+$chr-user-menu-line-height: inherit;
+$chr-user-menu-minify-breakpoint: 1250px;
```

Comment:
why not use the value for `$crm-menu-responsive-breakpoint` defined in this same variable file?